components/browser_kit.rst

```diff
@@ -109,6 +109,9 @@ method (which makes the needed HTTP POST request to submit the form contents)::
     $form['login'] = 'symfonyfan';
     $form['password'] = 'anypass';
 
+    // To upload a file, the value should be the filename
```

Comment:
`..., the value should be the filename` -> `..., the value should be the absolute file path` ?  Otherwise, we'd need to change the `__FILE__` example because that doesn't return only the filename.

Reply:
Fixed. Thanks